test/pulp3_plugin_integration_test.rb

```diff
@@ -0,0 +1,31 @@
+require 'test_helper'
+require 'json'
+require 'root/root_v2_api'
+require 'smart_proxy_pulp_plugin/pulp3_plugin'
+require 'smart_proxy_pulp_plugin/pulp3_client'
+
+class Pulp3FeaturesTest < Test::Unit::TestCase
+  include Rack::Test::Methods
+
+  def app
+    Proxy::PluginInitializer.new(Proxy::Plugins.instance).initialize_plugins
+    Proxy::RootV2Api.new
+  end
+
+  def test_features
+    Proxy::DefaultModuleLoader.any_instance.expects(:load_configuration_file).with('pulp.yml').returns(enabled: true, pulp_url: 'http://foo.com/foo')
+    Proxy::DefaultModuleLoader.any_instance.expects(:load_configuration_file).with('pulpnode.yml').returns(enabled: true, pulp_url: 'http://foo.com/foo')
+    Proxy::DefaultModuleLoader.any_instance.expects(:load_configuration_file).with('pulp3.yml').returns(enabled: true, pulp_url: 'http://foo.com/foo')
```

Comment:
Could you change this to `http://pulp3.example.com/foo` (and use example.com in other tests as well)? That guarantees we're reading the correct settings file as well.

Reply:
Good idea!.  done!